walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,9 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String whitespaceRemoved = name.replaceAll(" ", "");
+    String fixedName = whitespaceRemoved.replaceAll("[^a-zA-Z0-9]", "");
```

Comment:
```suggestion
    String fixedName = name.replaceAll("[^a-zA-Z0-9]\\s*", "");
```
I believe this suggestion should work, but feel free to keep it as is.

Reply:
it did work! :)
thank you